addons/binding/org.openhab.binding.atlona/README.md

```diff
@@ -21,20 +21,21 @@ If "Telnet Login" is enabled ("Network" tab from the switch configuration UI), y
 
 ## Binding configuration
 
-```
+```java
 atlona:pro3-88m:home [ ipAddress="192.168.1.30", userName="me", password="12345", polling=600, ping=30, retryPolling=10 ]
 ```
 
--   ipAddress: Hostname or IP address of the matrix switch
--   userName: (optional) the username to login with (only if Telnet Login is enabled)
--   password: (optional) the password to login with (only if Telnet Login is enabled)
--   polling: (optional) the time (in seconds) to poll the state from the actual switch (default: 600)
--   ping: (optional) the time (in seconds) to ping the switch to keep our connection alive (default: 30)
--   retryPolling: (optional) the time (in seconds) to retry a connection if the connection has failed (default: 10)
+- `ipAddress`: Hostname or IP address of the matrix switch
+- `userName`: (optional) the username to login with (only if Telnet Login is enabled)
+- `password`: (optional) the password to login with (only if Telnet Login is enabled)
+- `polling`: (optional) the time (in seconds) to poll the state from the actual switch (default: 600)
+- `ping`: (optional) the time (in seconds) to ping the switch to keep our connection alive (default: 30)
+- `retryPolling`: (optional) the time (in seconds) to retry a connection if the connection has failed (default: 10)
 
 ### username/password
 
-The userName/password configuration options are optional and are only required if you have your switch set with "Telnet Login" enabled (on the "Network" tab from the switch configuration UI).  The user must be a valid user listed on the "Users" tab of the switch configuration UI in this case.
+The userName/password configuration options are optional and are only required if you have your switch set with "Telnet Login" enabled (on the "Network" tab from the switch configuration UI).
+The user must be a valid user listed on the "Users" tab of the switch configuration UI in this case.
```

Comment:
Consider changing to, "`userNAME` must be a valid user listed on the "Users" tab of..."